src/eoapi/raster/eoapi/raster/factory.py

```diff
@@ -46,3 +55,192 @@ def stac_demo(
                 },
                 media_type="text/html",
             )
+
+
+class Infos(BaseModel):
+    """Response model for /list endpoint."""
+
+    searches: List[model.Info]
+    links: Optional[List[model.Link]]
+    numberMatched: Optional[int]
```

Comment:
Should we return numberMatched and numberReturned as a [context](https://raw.githubusercontent.com/radiantearth/stac-api-spec/v1.0.0-beta.4/fragments/context/json-schema/schema.json) object to be consistent with the stac spec?